src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,138 @@
+import React = require('react');
+import {EventHandler, ReactElement, SyntheticEvent} from "react";
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+const style = require('./radio-group.css');
+
+export interface RadioGroupProps {
+    children: JSX.Element[];
```

Comment:
This here to prevent typescript errors. If I remove it, it complains whenever I use props.children.